helpers.py

```diff
@@ -62,3 +63,21 @@ def get_cancellers(self) -> Set:
 
     def add_canceller(self, canceller: str) -> bool:
         return self.db.sadd(Key.CANCELLERS, canceller)
+    
+    def add_inv(self, author: int, inv: str):
```

Comment:
```suggestion
    def add_inv(self, author: int, inv: str) -> bool:
```

DB operations return a boolean based on success/failure of the operation.